edx_lint/files/common_constraints.txt

```diff
@@ -16,13 +16,24 @@
 Django<2.3
 
 # latest version is causing e2e failures in edx-platform.
+# See pyjwt[crypto]<2.0.0 comment.
 drf-jwt<1.19.1
 
-# Newer versions causing tests failures in multiple repos.
-pyjwt[crypto]==1.7.1
+# 4.0.0 requires pyjwt[crypto] 2.1.0. See pyjwt[crypto]<2.0.0 comment.
+edx-auth-backends<4.0.0
 
-# latest version requires PyJWT>=2.0.0 but drf-jwt requires PyJWT[crypto]<2.0.0,>=1.5.2
-social-auth-core<4.0.3
+# 7.0.0 requires pyjwt[crypto] 2.1.0. See pyjwt[crypto]<2.0.0 comment.
+edx-drf-extensions<7.0.0
+
+# PyJWT[crypto] 2.0.0 has a number of breaking changes that we are
+# actively working to fix. A number of the active constraints are all related
+# to this effort. Additionally, your IDA/service may also be affected directly
+# by these changes. You should not upgrade without knowing what you are doing.
+pyjwt[crypto]<2.0.0
```

Comment:
**Note to Reviewer:** This constraint should be equivalent to what was here, but a little looser. Also added more comments.